From: Ross Lagerwall <ross.lagerwall@citrix.com>
Subject: Skip call to flask

XenServer uses silo mode rather than flask so the toolstack should not attempt
to blindly make flask calls. This causes noise from the hypercall filter layer
in the dom0 kernel since xsm_op is blocked.

Signed-off-by: Ross Lagerwall <ross.lagerwall@citrix.com>
diff --git a/tools/libs/light/libxl_domain.c b/tools/libs/light/libxl_domain.c
index 6751fc785f72..f84dcf6762fe 100644
--- a/tools/libs/light/libxl_domain.c
+++ b/tools/libs/light/libxl_domain.c
@@ -276,14 +276,10 @@ void libxl__xcinfo2xlinfo(libxl_ctx *ctx,
                           const xc_domaininfo_t *xcinfo,
                           libxl_dominfo *xlinfo)
 {
-    size_t size;
-
     memcpy(&(xlinfo->uuid), xcinfo->handle, sizeof(xen_domain_handle_t));
     xlinfo->domid = xcinfo->domain;
     xlinfo->ssidref = xcinfo->ssidref;
-    if (libxl_flask_sid_to_context(ctx, xlinfo->ssidref,
-                                   &xlinfo->ssid_label, &size) < 0)
-        xlinfo->ssid_label = NULL;
+    xlinfo->ssid_label = NULL;
 
     xlinfo->dying      = !!(xcinfo->flags&XEN_DOMINF_dying);
     xlinfo->shutdown   = !!(xcinfo->flags&XEN_DOMINF_shutdown);
